### PR TITLE
updated look of quest-menu and culprit-menu

### PR DESCRIPTION
**Pull request by adaj-1**
- Rearranged the menu buttons and made sure they were selectable over the whole button surface area.
- Added a quest status in the quest-menu to help players identify how many quests are left in the game.
- Updated the look of the culprit-menu to match the other menus

https://user-images.githubusercontent.com/67992220/204112472-837a37e8-a559-4d70-aec8-c6c285323942.mp4

Note: Still having issues determining how to hide buttons that load in from another scene. An example of this is the Culprits, Left, Right, and Rooms button show in item-menu. Left and right buttons are able to be triggered while in a menu. This may be some sort of regression as I thought we had previously fixed this. Will look into this with my time later today and tomorrow

### CODE REVIEW

File: src/default_env.tres [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
-sky_energy = 0.78
-ground_bottom_color = Color( 0.145098, 0.227451, 0.0470588, 1 )
-ground_horizon_color = Color( 0.0392157, 0.0509804, 0.0470588, 1 )
-sun_color = Color( 0.0862745, 0, 0, 1 )
-sun_latitude = -29.33
-sun_longitude = -41.98
-sun_angle_min = 53.41
-sun_angle_max = 90.83
-texture_size = 1
+sky_horizon_color = Color( 0.160784, 0.109804, 0.207843, 1 )
```
Comment (anson-li) [inline src/default_env.tres]: Just wondering, why were the default_env values for the sky changed? Could you please add a before/after if this is intentional? Just wondering as this was edited for the study room + for the garden rooms previously

File: src/autoloads/quests.gd [hunk 2]
```diff
@@ -10,6 +12,7 @@ func remove_quest(remove: PlayerQuest) -> void:
     for quest in quest_list:
         if quest.label == remove.label:
             quest_list.erase(quest);
+            completed_quests += 1;
```
Comment (anson-li) [inline src/autoloads/quests.gd:15]: Do we need to consider a stopgap - eg. is there any way for total_quests to hit > 5 and cause issues in the game?
```suggestion
            completed_quests += 1;
            completed_quests = min(completed_quests, 5)
```

**Review by anson-li — COMMENTED** [review]
Looks good, just a couple of minor questions. Please consider any differences with your PR and Bryden's PR as they may cover some shared areas.

File: src/default_env.tres (hunk 1 above)
Comment (adaj-1) [reply src/default_env.tres]: hmm not sure why this was changed I thought I removed this from the commit. Theres no need for this change for the work here.

File: src/autoloads/quests.gd (hunk 2 above)
Comment (adaj-1) [reply src/autoloads/quests.gd:15]: fixed this up in newest commit!

**Review by brydenli — APPROVED** [review]

**Pull request merged**